CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 ## [Unreleased]
 
+## [1.0.7] - 2022-12-12
+
```

Comment:
Normally, bullets under the `unreleased` section are moved here. Can you edit this and add bullets to describe the changes.

Reply:
Done